packages/client/hmi-client/src/components/pyciemss/tera-pyciemss-output-settings.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+	<section class="form-section">
+		<h5>
+			Output settings
+			<i v-tooltip="outputSettingsToolTip" class="pi pi-info-circle" />
+		</h5>
+
+		<!--Summary-->
+		<h5>Summary</h5>
+		<tera-checkbox
+			v-model="knobs.summaryCheckbox"
+			inputId="generate-summary"
+			label="Auto-generate operation summary"
+			subtext="Automatically generates a brief summary of the inputs and outputs."
+			disabled
+			@update:model-value="emit('update-self', knobs)"
+		/>
+
+		<!-- Note that this only exists for optimize. -->
+		<div v-if="props.successDisplayChartsCheckbox">
+			<!--Success Criteria-->
+			<Divider />
+			<h5>
+				Success Criteria
+				<i v-tooltip="outputSettingsToolTip" class="pi pi-info-circle" />
+			</h5>
+			<tera-checkbox
+				v-model="knobs.successDisplayChartsCheckbox"
+				inputId="success-criteria-display-charts"
+				label="Display chart(s)"
+				subtext="Turn this on to generate an interactive chart of the success criteria conditions."
+				disabled
+				@update:model-value="emit('update-self', knobs)"
+			/>
+		</div>
+
+		<div v-if="interventionsOptions.length > 0">
+			<Divider />
+			<!--Interventions-->
+			<h5>
+				Interventions
+				<i v-tooltip="outputSettingsToolTip" class="pi pi-info-circle" />
+			</h5>
+			<MultiSelect
+				v-model="knobs.selectedInterventionVariables"
+				:options="props.interventionsOptions"
+				placeholder="What do you want to see?"
+				filter
+				@update:modelValue="emit('update-self', knobs)"
+			/>
+			<tera-checkbox
+				v-model="knobs.interventionsDisplayChartsCheckbox"
+				inputId="interventions-display-charts"
+				label="Display chart(s)"
+				disabled
+				@update:model-value="emit('update-self', knobs)"
+			/>
+		</div>
+		<Divider />
+
+		<!--Simulation plots-->
+		<h5>
+			Simulation plots
+			<i v-tooltip="outputSettingsToolTip" class="pi pi-info-circle" />
+		</h5>
+		<MultiSelect
+			v-model="knobs.selectedSimulationVariables"
+			:options="props.simulationChartOptions"
+			placeholder="What do you want to see?"
+			filter
+			@update:modelValue="emit('update-self', knobs)"
+		/>
+		<tera-checkbox
+			v-model="knobs.simulationDisplayChartsCheckbox"
+			inputId="sim-plots-display-charts"
+			label="Display chart(s)"
+			disabled
+			@update:model-value="emit('update-self', knobs)"
+		/>
+	</section>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import TeraCheckbox from '@/components/widgets/tera-checkbox.vue';
+import Divider from 'primevue/divider';
+import MultiSelect from 'primevue/multiselect';
+
+const props = defineProps<{
+	successDisplayChartsCheckbox?: boolean;
+	summaryCheckbox?: boolean;
+	interventionsDisplayChartsCheckbox?: boolean;
+	simulationDisplayChartsCheckbox?: boolean;
+	selectedSimulationVariables?: string[];
+	selectedInterventionVariables?: string[];
+	simulationChartOptions: string[];
+	interventionsOptions: string[];
+}>();
```

Comment:
Do we have the design sketches you can either link to? or post a few screen grabs here? 

I am wondering if we should assess this a bit more systematically ... I am thinking if we can pass in the `<State>` into these component and it figures it out? or if each operator should define some kind of emit-function of the things it wants to be configured. This component right now feels a bit like a free-for-all and not entire sure what is what.

Reply:
Chatted with Tom off thread:

There is a pretty high chance we'd need to overhaul the output settings panel and how things work:
- Some of the design elements are problematic, not really up-to-date with the latest thinking, or are some what contradictory. For example we'd need to think about how these settings are going to work with reporting features
- Likewise, the "chartConfig" is getting rejigged to take into consideration annotations, this will impact how we would pass things back and forth between the operator drilldown and the settings panel as they wouldn't be just variable string arrays
- It would still be good to assess the situation in a more systematic manner. It may be nice, if possible, for each operator to provide a generator function of its output-settings, and how the settings should be displayed.